api/XMLDocument.json

```diff
@@ -50,6 +50,87 @@
           "deprecated": false
         }
       },
+      "async": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/XMLDocument/async",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": [
+              {
+                "version_added": "68",
+                "flags": [
+                  {
+                    "name": "dom.xmldocument.async.enabled",
+                    "type": "preference",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": "See <a href='https://bugzil.la/1546112' title='Remove the code for XMLDocument.load/async if possible'>bug 1546112</a> for complete removal."
```

Comment:
Can we get rid of the `title` attributes in here?  I don't really think they belong, especially since the bug explains it all already.  (I'll only be commenting on this one.)

Reply:
The `title` attributes were in this file before and I&nbsp;don't think having them hurts anything.

Also it passes the linter.

Reply:
If that’s the case, we should have a linter where only the `href` attribute is on `<a>` tags.